docs/Troubleshooting.md

```diff
@@ -95,3 +95,20 @@ If you run into issues where running `react-native init` hangs in your system, t
 ```
 react-native init --verbose
 ```
+
+## Unable to execute JS call: __fbBatchedBridge is undefined
+Refer to issue [#6282](https://github.com/facebook/react-native/issues/6282)
+
+In `node_modules/react-native/packager/react-native-xcode.sh` comment out the following lines:
```

Comment:
I think that the `__fbBatchedBridge` error can happen any time the JavaScript fails to be bundled for whatever reason. So while this may fix a specific issue, it may not solve every single case of this error showing up.

Given this, I'm not sure what's the best place to document it aside from existing GitHub issues threads.